### PR TITLE
not pretty but does work

### PR DESCRIPTION
**Pull request by birdage**

### CODE REVIEW

**Comment by birdage** [comment]
@oceanzus silly monkey!

File: ooiservices/app/uframe/controller.py [hunk 1]
```diff
@@ -360,6 +363,7 @@ def get_uframe_stream_contents(mooring, platform, instrument, stream_type, strea
             query = '?beginDT=%s&endDT=%s&execDPA=true' % (start_time, end_time)
         UFRAME_DATA = current_app.config['UFRAME_URL'] + current_app.config['UFRAME_URL_BASE']
         url = "/".join([UFRAME_DATA,mooring, platform, instrument, stream_type, stream + query])     
+        print url
```
Comment (oceanzus) [inline ooiservices/app/uframe/controller.py]: I'm silly?

File: ooiservices/app/uframe/controller.py [hunk 2]
```diff
@@ -323,12 +325,13 @@ def get_uframe_instrument_metadata(ref):
...
-        response = requests.get(url)
+  
+        response = requests.get(url)      
         if response.status_code == 200:
-            data = response.json
+            data = response.json()
             return jsonify(metadata=data['parameters'])
         return jsonify(metadata={}), 404
-    except:
+    except Exception,e:
```
Comment (oceanzus) [inline ooiservices/app/uframe/controller.py]: except Exception as e:

**Comment by oceanzus** [comment]
https://github.com/birdage/ooi-ui-services/pull/1

**Comment by oceanzus** [comment]
Killed my PR against yours.  Merging to test, will fix it up in another PR if needed.

**Comment by birdage** [comment]
sounds good!

**Pull request merged**